packages/components/src/components/NFTGallery/NFTGallery.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import fetch from 'cross-fetch';
+
+import { VStack, Heading, Grid } from '@chakra-ui/react';
+import { NFT } from '../NFT';
+
+export interface NFTGalleryProps {
+  address: string;
+  gridWidth?: number;
+}
+
+export interface OpenSeaAsset {
+  token_id: string;
+  image_url: string;
+  name: string | null;
+  asset_contract: {
+    name: string;
+    symbol: string;
+  };
+}
+
+/**
+ * @dev Component to display a grid of NFTs owned by an address. It uses the OpenSea API to fetch
+ * the NFTs.
+ */
+export const NFTGallery = ({ address, gridWidth = 4 }: NFTGalleryProps) => {
+  const [nfts, setNfts] = React.useState<OpenSeaAsset[]>([]);
+
+  useEffect(() => {
+    fetch(`https://api.opensea.io/api/v1/assets?owner=${address}`)
+      .then((res) => res.json())
+      .then((data) => setNfts(data.assets));
+  }, []);
```

Comment:
Shouldn't `address` be inside the dependencies-array? In case someone switches his active account on metamask for example.

Could be I'm wrong, not quite that familar on how it works yet.

Reply:
you're 100% right. thanks for catching this